backend/qemu.pm

```diff
@@ -23,7 +23,7 @@ use autodie ':all';
 use base 'backend::virt';
 
 use File::Basename 'dirname';
-use File::Path 'mkpath';
+use File::Path qw(path mkpath);
```

Comment:
Isn't it supposed to use that function from the `Mojo::File` module?

Reply:
absolutely :)